api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -219,22 +220,24 @@
       (is (thrown-with-msg? ExceptionInfo #"clj-http: status 400"
                             (endpoints/exec-workload request))))))
 
-(def ^:private covid-workload-request
+(defn ^:private covid-workload-request []
   "Build a covid workload request."
-  (let [dataset "cd25d59e-1451-44d0-8a24-7669edb9a8f8"
-        column  "run_date"
-        table   "flowcells"
-        workspace-template "CDC_Viral_Sequencing"
-        workspace-uniqued  "GPc586b76e8ef24a97b354cf0226dfe583"
-        workspace-namespace  "wfl-dev"
-        workspace-name (str/join "_" [workspace-template workspace-uniqued])
-        workspace      (str/join "/" [workspace-namespace workspace-name])
-        mc-namespace   "cdc-covid-surveillance"
-        mc-name        "sarscov2_illumina_full"
-        methodConfiguration (str/join "/" [mc-namespace mc-name])
-        source   (util/make-map column dataset table)
-        executor (util/make-map methodConfiguration workspace)
-        sink     (util/make-map workspace)]
+  (let [source   {:name    "Terra DataRepo"
+                  :dataset "79fc88f5-dcf4-48b0-8c01-615dfbc1c63a"
+                  :table   "flowcells"
+                  :column  "last_modified_date"}
+        executor {:name                       "Terra"
+                  :workspace                  "wfl-dev/CDC_Viral_Sequencing"
+                  :methodConfiguration        "cdc-covid-surveillance/sarscov2_illumina_full"
+                  :methodConfigurationVersion 1
+                  :fromSource                 "importSnapshot"}
+        sink     {:name           "Terra Workspace"
+                  :workspace      "wfl-dev/CDC_Viral_Sequencing"
+                  :table          "flowcells"
+                  :identifier     "flowcell_id"
```

Comment:
curious question, do we want to add an `entity` here?

Reply:
thanks. fixed.